src/sentry/rules/conditions/tagged_event.py

```diff
@@ -35,7 +35,16 @@ class MatchType(object):
 class TaggedEventForm(forms.Form):
     key = forms.CharField(widget=forms.TextInput())
     match = forms.ChoiceField(MATCH_CHOICES.items(), widget=forms.Select())
-    value = forms.CharField(widget=forms.TextInput())
+    value = forms.CharField(widget=forms.TextInput(), required=False)
+
+    def clean(self):
+        super(TaggedEventForm, self).clean()
+
+        match = self.cleaned_data.get("match")
+        value = self.cleaned_data.get("value")
+
+        if match != MatchType.IS_SET and match != MatchType.NOT_SET and not value:
+            raise forms.ValidationError("This field is required.")
```

Comment:
`"The 'value' field is required"`

Reply:
It's this because of https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/rest_framework/rule.py#L49